packages/istanbul-lib-report/lib/tree.js

```diff
@@ -23,8 +22,10 @@ const util = require('util');
  *  and nothing else.
  * @constructor
  */
-function Visitor(delegate) {
-    this.delegate = delegate;
+class Visitor {
+    constructor(delegate) {
+        this.delegate = delegate;
+    }
 }
 
 ['Start', 'End', 'Summary', 'SummaryEnd', 'Detail'].forEach(k => {
```

Comment:
yikes :\

Can we do this with `Object.defineProperty` instead at least?

Reply:
Done